sources/packages/api/src/main.ts

```diff
@@ -6,6 +6,11 @@ async function bootstrap() {
   const app = await NestFactory.create<NestExpressApplication>(AppModule);
   app.setGlobalPrefix('api')
   const port = process.env.PORT || 3000;
+
+  if(process.env.NODE_ENV !== 'production'){
```

Comment:
I think CROS should enable for all env. Because our api will be exposed through api gateway, so any third party app can access them future.

Reply:
I would prefer to keep it as locked as possible for now and enable it when the time comes.
All discussions about API Gateway seem to be too initial and I am still not sure if this specific API would be exposed.